test/__snapshots__/plugin.test.js.snap

```diff
@@ -34,7 +33,6 @@ Array [
     },
     "packageManager": "npm",
     "prompt": true,
-    "quiet": false,
```

Comment:
I will look at other repos where this plugin are using, I think better to keep `quiet` for better DX, let's move it in `packageManager.options.quiet` (just put `--silent`/`'--silent', '--no-progress'`) in arguments, also let's add pnpm support

Reply:
Updated. It seems `pnpm` don't have `--silent` flag for the [`add` comman](https://pnpm.io/cli/add#options).

Reply:
some options are global, some only for commands, `--reporter=silent` should be global you can test it locally